contracts/alice/Makefile

```diff
@@ -6,3 +6,5 @@ wasm: ## Generate the optimized WASM for the contract given
     		-Z build-std=core,alloc,panic_abort \
     		-Z build-std-features=panic_immediate_abort \
     		--target wasm32-unknown-unknown
+
+test:
```

Comment:
These test rules are empty so recursive make can work properly when touching these folders as well. Since they don't have really have tests - they're used elsewhere - I thought it appropriate.